xml_converter/generators/code_generator.py

```diff
@@ -289,12 +303,14 @@ class AttributeVariable:
     xml_fields: List[str]
     protobuf_field: str
     args: List[str] = field(default_factory=list)
-    default_xml_fields: List[str] = field(default_factory=list)
-    xml_export: str = ""
+    default_xml_field: str = ""
     side_effects: List[str] = field(default_factory=list)
-    compound_name: Optional[str] = None
+    xml_parent_export: List[str] = field(default_factory=list)
+    parser_flag_name: Optional[str] = ""
```

Comment:
`parser_flag_name` might be a bit too generic. This is the value `a` which allows us to write `a.b = value` instead of `b = value`.

Reply:
changed to `attribute_flag_name`